src/index.ts

```diff
@@ -1,24 +1,27 @@
 import { readFile } from 'node:fs/promises';
 import type { PoolClient, QueryResult } from 'pg';
+import {
+  ReadDictionaryResult,
+  ReadSequenceResult,
+  ReadSequenceListResult,
+  WriteSequenceResult,
+  ReadParcelResult,
+} from './types/db-types';
+import { dictionaryQuery, queryReadParcel } from './helpers/db-helpers';
+import { Config } from './types';
 export * from './types';
 
-// types and helpers for making DB queries
-
-// type for results of the Sequence List db query
-export type SequenceListResult = {
-  name: string;
-  id: number;
-  workspace_id: number;
-  parcel_id: number;
-  owner?: string;
-  created_at: string;
-  updated_at: string;
-};
-
+/**
+ * Reads a list of sequences for a given `workspaceId`.
+ *
+ * @param dbClient - A client that is part of our connection pool.
+ * @param workspaceId - The id of the workspace the sequence is a part of.
+ * @returns {@link ReadSequenceListResult}
```

Comment:
Since we're using TS + **TSDoc** instead of **JSDoc** here, any doc comment that specifies a return type is redundant and risks getting out of sync, since the TS type system is the source of truth here (see https://tsdoc.org/pages/tags/returns/ ). 

I'm going to replace these with descriptions of the return values